tests/Form/Type/ModelTypeTest.php

```diff
@@ -49,13 +52,26 @@ public function testGetDefaultOptions(): void
         $this->assertNull($options['class']);
         $this->assertNull($options['property']);
         $this->assertNull($options['query']);
-        $this->assertCount(0, $options['choices']);
+        $this->assertEquals($choices, $options['choices']);
```

Comment:
The CS checks forbid this call. If you can't use `assertSame()` because there is an object or something that can't be the same as the expectation, try to use a different approach.